modules/caddyhttp/reverseproxy/circuitbreaker.go

```diff
@@ -130,9 +130,14 @@ func (c *localCircuitBreaker) checkAndSet() {
 
 // Config represents the configuration of a circuit breaker.
 type Config struct {
+	// The threshold over sliding window that would trip the circuit breaker
 	Threshold float64 `json:"threshold"`
-	Type      string  `json:"type"`
-	TripTime  string  `json:"trip_time"`
+	// Possible values: latency, error_ratio, and status_ratio. It
+	// defaults to latency.
+	Type string `json:"type"`
```

Comment:
Thinking on it more, I actually really like your idea to rename `type` to `factor`.

Reply:
Done